internal/services/nginx/nginx_configuration_resource_test.go

```diff
@@ -255,3 +296,200 @@ resource "azurerm_nginx_deployment" "test" {
 }
 `, data.RandomInteger, data.Locations.Primary)
 }
+
+func (a ConfigurationResource) httpsTemplate(data acceptance.TestData) string {
```

Comment:
if you did `CertificateResource.basic` in test pkg, I think you can safely drop all of this test-setup code and just have the config that references that cert.

Reply:
Done!